packages/@aws-cdk/aws-codebuild/test/test.project.ts

```diff
@@ -1466,6 +1466,124 @@ export = {
 
         test.done();
       },
+
+      'can be provided as a SecretValue for SecretsManager'(test: Test) {
```

Comment:
This description needs to be better. Right now, it doesn't describe anything about the test. Something like

```suggestion
      'can be provided as a SecretValue from a SecretsManager imported Secret by complete ARN'(test: Test) {
```